### PR TITLE
DI-257 Add latency metric

### PR DESCRIPTION
**Pull request by ed-sparkes**
## Link to JIRA Ticket

- https://nhsd-jira.digital.nhs.uk/browse/DI-257

## Description

Pass event received to event sender so that difference between an event received and change accepted can be logged as latency. 
Use embedded metric logger to save metric to cloudwatch, using environment as dimension.

## Type of change

Delete not appropriate

- New feature (non-breaking change which adds functionality)
- 
## Testing

Please tick the testing that has been completed

- [x] Unit tests
- [ ] Component tests

## Developer Checklist

- [x] I have performed a self-review of my own code
- [x] I have run the [code formatting checks](../README.md#code-quality)
- [x] I have run the [code quality checks](../README.md#code-quality)
- [x] New code meets [standards](https://nhsd-confluence.digital.nhs.uk/display/DI/DI+Ways+of+Working) agreed by the team
- [x] Unit test code coverage is at or above 80%
- [x] New and existing unit tests pass locally with my changes
- [ ] Tests have added that prove my fix is effective or that my feature works (Component tests)
- [ ] I have made corresponding changes to the documentation
- [ ] I have cleaned down my environment (if created)

## Code Reviewer Checklist

- [ ] I have run the unit tests and they run correctly
- [ ] I can confirm the changes have been tested or approved by a tester
- [ ] I can confirm no remaining infrastructure is left over from this branch

### CODE REVIEW

File: application/event_sender/event_sender.py [hunk 1]
```diff
@@ -26,6 +30,12 @@ def lambda_handler(event: Dict[str, Any], context: LambdaContext) -> None:
         "Received change request",
         extra={"change_request": body["change_payload"], "correlation_id": logger.get_correlation_id()},
     )
+    message_received = body["message_received"]
+
     change_request = ChangeRequest(body["change_payload"])
     change_request.post_change_request()
+    now_ms = time_ns() // 1000000
```
Comment (JackPlowman) [inline application/event_sender/event_sender.py:37]: Is this milliseconds or nanoseconds? As we are using milliseconds elsewhere is this a problem for the latency calculation.
https://www.geeksforgeeks.org/python-time-time_ns-method/

**Review by JackPlowman — CHANGES_REQUESTED** [review]

**Review by JackPlowman — DISMISSED** [review]
LGTM 👍

**Review by rohith1122 — DISMISSED** [review]
LGTM.

**Review by JackPlowman — APPROVED** [review]
LGTM 👍

**Review by rohith1122 — APPROVED** [review]

**Pull request merged**